src/test/fixtures/datasources.ts

```diff
@@ -83,6 +83,9 @@ export const METRICS_DATASOURCE: DataSourceInstanceSettings<any> & WithAccessCon
       grafanaDependency: '',
       grafanaVersion: '*',
       plugins: [],
+      extensions: {
+        exposedComponents: [],
+      },
```

Comment:
Just wondering, but why was this needed?

Reply:
Typescript